### DISCUSSION TITLE
Unreadable Firebase ML Kit Text Recognition Project

### DISCUSSION THREAD

**Discussion by AdityaWiradarma** [community/community#49507] [Mobile]
### Select Topic Area

Product Feedback

### Body

I made a license plate scan application with the vehicle, for coding it will run smoothly and there are no errors, but when scanning the vehicle plate the plate number doesn't want to appear, for the project using Firebase ML Kit Text Recognition

https://github.com/AdityaWiradarma/aplikasi-scan-plat-nomor.git

**Comment by eliperkins** [comment]
Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as https://github.com/googlesamples/mlkit. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
